modules/launcher/src/test/java/org/mule/runtime/module/launcher/MuleContainerTestCase.java

```diff
@@ -41,15 +42,16 @@
 
     private MuleCoreExtensionManagerServer coreExtensionManager;
 
-    private DeploymentService deploymentService;
+    private DeploymentService deploymentService = mock(DeploymentService.class);
+
+    private RepositoryService repositoryService = mock(RepositoryService.class);
 
     @Before
     public void setUp() throws Exception
     {
         coreExtensionManager = mock(MuleCoreExtensionManagerServer.class);
-        deploymentService = mock(DeploymentService.class);
 
-        container = new MuleContainer(deploymentService, coreExtensionManager);
+        container = new MuleContainer(deploymentService, repositoryService, coreExtensionManager);
```

Comment:
Add tests for the repositoryService usage (similar to how deploymentService is tested)